### PR TITLE
Add new chapter

### PR DESCRIPTION
**Pull request by leonardcser**
Below are the changes mentioned in the issue #20.

Note that theses changes depend on the merge request #17 for the download path and filename of the google service account key.

### CODE REVIEW

File: docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# Cleaning up resources and environments
+
+Now that you have completed the guide, it's important to clean up the resources and environments you created to avoid incurring unnecessary costs. Here's a step-by-step guide to help you do that.
+
+!!! warning
+    If you are using a new google cloud account, make sure to delete the resources you created before the credits are consumed. Otherwise, **you will be charged for the resources you created.**
```
Comment (ludelafo) [inline docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md:6]: Rename "google cloud" to "Google Cloud" as it's a name.

File: docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+
```
Comment (ludelafo) [inline docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md:8]: You can remove unnecessary spaces.

File: docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+## Steps
+
+
```
Comment (ludelafo) [inline docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md:11]: You can remove unnecessary spaces.

File: docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
+# Cleaning up resources and environments
+
+Now that you have completed the guide, it's important to clean up the resources and environments you created to avoid incurring unnecessary costs. Here's a step-by-step guide to help you do that.
+
+!!! warning
```
Comment (ludelafo) [inline docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md:5]: You can add a space after the admonition for consistency.

File: docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+
+## Steps
+
+
+### Google Cloud
+
+#### Delete the Google Storage bucket
+
+!!! warning
```
Comment (ludelafo) [inline docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md:16]: You can add a space after the admonition for consistency.

File: docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+    Ignore this if you plan deleting the git repository. When deleting the Google Storage bucket, **it will break the DVC remote**. You will need to reconfigure it to a new remote if you want to keep the git repository to use DVC.
```
Comment (ludelafo) [inline docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md:17]: "git" -> "Git"

File: docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+
+To delete the Google Storage bucket you created:
```
Comment (ludelafo) [inline docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md:19]: Is it possible to use the Google Cloud CLI (gcloud) so we don't have to go through the UI?

File: docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+
+1. Go to the [Google Cloud Storage Console](https://console.cloud.google.com/storage){:target="_blank"}.
+2. Make sure you selected the correct project.
+3. Select the bucket you want to delete from the bucket list.
+4. Click on **Delete** at the top of the page.
+5. Follow the instructions to delete the bucket.
+
+#### Delete the Service Account
+
+To delete the service account you created:
```
Comment (ludelafo) [inline docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md:29]: Is it possible to use the Google Cloud CLI (gcloud) so we don't have to go through the UI?

File: docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+#### Delete the Google Cloud project
+
+To delete the Google Cloud project you created:
+
+1. Go to the [Google Cloud Ressource Manager Console](https://console.cloud.google.com/cloud-resource-manager){:target="_blank"}.
+2. Select the project you created.
+3. Click on **Delete** at the top of the page.
+4. Follow the instructions to shut down the project.
+
+
```
Comment (ludelafo) [inline docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md:54]: You can remove unnecessary spaces.

File: docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+### Repository
```
Comment (ludelafo) [inline docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md:55]: You can add a space after the title.

File: docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+=== ":simple-github: GitHub"
+
+	To delete the GitHub repository you created:
+
+    1. Go to the repository page in GitHub.
+    2. Click on **Settings** above the repository.
+    3. Scroll down to the "Danger Zone" section at the bottom of the page.
+    4. Click on **Delete this repository**.
+    5. Follow the instructions to delete the repository.
```
Comment (ludelafo) [inline docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md:64]: You can replace the spaces with tabulations.

File: docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+    5. Click on the **Delete project** button.
+    6. Follow the instructions to delete the repository.
+
+	To delete the GitLab Personal Access Token you created:
+
+    1. Go to your GitLab **Profile preferences** at the top right of the page.
+    2. Click on **Access Tokens** in the left sidebar.
+    3. Find the Personal Access Token you created for this guide.
+    4. Click on the **Revoke** button next to it.
+    5. Follow the instructions to revoke the token.
```
Comment (ludelafo) [inline docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md:83]: You can replace the spaces with tabulations.

File: docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+
+
```
Comment (ludelafo) [inline docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md:85]: You can remove unnecessary spaces.

File: docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+### Clean up your local environment
+
+To clean up your local environment, you can simply delete the project directory you created for this guide.
+
+
```
Comment (ludelafo) [inline docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md:90]: You can remove unnecessary spaces.

File: docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md [hunk 15]
```diff
@@ -0,0 +1,110 @@
...
+- Google Cloud project
+- Google Cloud Storage bucket
+- Google Cloud Service Account
+- Google Cloud Service Account key
+- GitHub/GitLab repository
+- If you used GitLab, you also created a Personal Access Token
+- Project directory on your local machine
+
+
```
Comment (ludelafo) [inline docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md:105]: You can remove unnecessary spaces.

File: mkdocs.yml [hunk 16]
```diff
@@ -27,6 +27,7 @@ nav:
       - "Chapter 6: Orchestrate the workflow with a CI/CD pipeline": the-guide/chapter-6-orchestrate-the-workflow-with-a-cicd-pipeline/index.md
       - "Chapter 7: Track model evolutions in the CI/CD pipeline with CML": the-guide/chapter-7-track-model-evolutions-in-the-cicd-pipeline-with-cml/index.md
       - "Chapter 8: Serve the model with MLEM": the-guide/chapter-8-serve-the-model-with-mlem/index.md
+      - "Chapter 9: Cleaning up resources and environments": the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md
```
Comment (ludelafo) [inline mkdocs.yml:30]: As some of the cleanup is already in the "Conclusion" chapter, what if we renamed the chapter "Conclusion" to "Clean up and Conclusion" and merge these chapters together? What do you think?

**Review by ludelafo — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 16 above)
Comment (leonardcser) [reply mkdocs.yml]: It makes much more sense to merge both.

**Review by ludelafo — APPROVED** [review]
All good to me!

**Pull request merged**